### PR TITLE
feat(PNI): include PNI Database

### PR DESCRIPTION
**Pull request by luabida**

### CODE REVIEW

File: poetry.lock
Comment (luabida) [inline poetry.lock:1]: for some reason, poetry.lock wasn't on repo yet 🤔

File: pyproject.toml [hunk 1]
```diff
@@ -22,7 +22,7 @@ wget = "^3.2"
 loguru = "^0.6.0"
 Unidecode = "^1.3.6"
 dateparser = "^1.1.8"
-pandas = ">=1.5.3"
```
Comment (luabida) [inline pyproject.toml:25]: pandas 1.5.3 uses applymap instead of map

**Review by esloch — APPROVED** [review]
Good job!

File: pysus/ftp/databases/pni.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            "vacinado, agregados por faixa etária, período de tempo e área geográfica. "
+            "Possibilita também o controle do estoque de imunobiológicos necessário "
+            "aos administradores que têm a incumbência de programar sua aquisição e "
+            "distribuição. Controla as indicações de aplicação de vacinas de "
+            "imunobiológicos especiais e seus eventos adversos, dentro dos Centros "
+            "de Referências em imunobiológicos especiais."
+        ),
+    }
+    groups = {
+        "CPNI": "Centro de Parto Normal Intra-Hospitalar",
```
Comment (fccoelho) [inline pysus/ftp/databases/pni.py:32]: It's curious that PNI has a dataset for "Parto Normal Intra-Hospitalar", does not seem related...

File: pysus/ftp/databases/sim.py [hunk 3]
```diff
@@ -6,24 +6,29 @@
...
         Directory("/dissemin/publicos/SIM/CID9/DORES"),
-    ]
+    )
     metadata = {
         "long_name": "Sistema de Informação sobre Mortalidade",
         "source": "http://sim.saude.gov.br",
         "description": "",
     }
-    groups = {"CID10": "DO", "CID9":"DOR"}
+    groups = {"CID10": "DO", "CID9": "DOR"}
```
Comment (fccoelho) [inline pysus/ftp/databases/sim.py:18]: Is this information visible to the user? Because CID10 and CID9 (ICD 10 and 9, in english) are well-known to epidemiologists, DO and DOR are just meaningless acronyms, AFAIK.

File: pysus/online_data/CIHA.py [hunk 4]
```diff
@@ -8,9 +8,26 @@
...
-from pysus.online_data import FTP_Downloader
+from pysus.ftp.databases.ciha import CIHA
 from pysus.ftp import CACHEPATH
 
+ciha = CIHA().load()
+
+
+def get_available_years(
+    states: Union[list, str] = None,
+    months: Union[str, int, list] = None
```
Comment (fccoelho) [inline pysus/online_data/CIHA.py:19]: I don't think it makes sense to specify months here

File: pysus/online_data/CNES.py [hunk 5]
```diff
@@ -22,6 +24,36 @@
 }
 
 
+def get_available_years(
+    group: str,
+    states: Union[str, list] = None,
+    months: Union[str, list, int] = None,
```
Comment (fccoelho) [inline pysus/online_data/CNES.py:30]: Same comment as before. and for the other modules

**Review by fccoelho — CHANGES_REQUESTED** [review]

File: pysus/ftp/databases/pni.py (hunk 2 above)
Comment (luabida) [reply pysus/ftp/databases/pni.py:32]: By any chance, do you know the difference between CPNI and DPNI? both file prefixes are found in the ftp server

File: pysus/ftp/databases/sim.py (hunk 3 above)
Comment (luabida) [reply pysus/ftp/databases/sim.py:18]: I use the acronyms to differentiate between CID9 and CID10 in the file formatting

**Comment by luabida** [comment]
@fccoelho This PR needs merge in order to continue https://github.com/AlertaDengue/episcanner-downloader/pull/13

**Review by fccoelho — APPROVED** [review]
